### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Pmutkure009**
Added Telegram Chat link at the Bottom.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,3 +92,6 @@ To learn React, check out the [React documentation](https://reactjs.org/).
 
 # License
 This project is licensed under the OpenMRS Public License, see the [license](LICENSE) file for details.
+
+# Telegram Chat
+Join [this Telegram chat](https://t.me/joinchat/EKjttk-RVASZbqN2xM1FFQ) if you are intreasted on working on this project.
```
Comment (sviho) [inline README.md]: Could you fix a spelling error here? You probably mean 'interested' :)

Comment (Pmutkure009) [reply README.md]: Yeah, I was just doing that. Done now

**Comment by ykarim** [comment]
Also, could you check the branch name and make it more appropriate. Check out OpenMRS Pull Request Tips. By the way, the intent of this was to create a badge for Telegram using shields.io and such similar to how it's done in the OpenMRS Android Client. Could you attempt at doing that and placing the badge at the top

File: README.md (hunk 1 above)
Comment (Bisht13) [reply README.md]: Could you squash the commits?

**Comment by ykarim** [comment]
And the pull request name is also vague. Could you make it more specific like add telegram link to readme

**Comment by Bisht13** [comment]
@Pmutkure009 Please refer to this https://wiki.openmrs.org/display/docs/Pull+Request+Tips for now I'll update the README if that's okay with you 😄

**Comment by Pmutkure009** [comment]
Ohh sorry!

I think I am quite a bit under experienced for this, I think @Bisht13 should do it :)

**Comment by ykarim** [comment]
That's totally fine we appreciate the effort though! And I think might be better if you were to take a shot at this, it does help to learn along the way. We could help you out

**Pull request closed without merging**